script/sync

```diff
@@ -134,6 +134,30 @@ def sync_ruby(version)
   puts "done."
 end
 
+def sync_node(version)
+  s3_key = "nodes/Darwin/#{s3_cellar}#{os}/#{version}.tar.bz2"
```

Comment:
Thanks!, I doesn't seems to be any references..
I did run this
```
find /opt/nodes/. -type f -print | xargs otool -L
```

This was the output
```
/opt/nodes/./0.10.36/bin/node:
	/System/Library/Frameworks/Carbon.framework/Versions/A/Carbon (compatibility version 2.0.0, current version 157.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1213.0.0)
	/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation (compatibility version 300.0.0, current version 1153.20.0)
	/System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices (compatibility version 1.0.0, current version 62.0.0)
	/System/Library/Frameworks/ApplicationServices.framework/Versions/A/ApplicationServices (compatibility version 1.0.0, current version 48.0.0)
	/usr/lib/libstdc++.6.dylib (compatibility version 7.0.0, current version 104.1.0)
	/usr/lib/libgcc_s.1.dylib (compatibility version 1.0.0, current version 283.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1153.18.0)
/opt/nodes/./0.10.36/include/node/ares.h: is not an object file
/opt/nodes/./0.10.36/include/node/ares_version.h: is not an object file
/opt/nodes/./0.10.36/include/node/common.gypi: is not an object file
/opt/nodes/./0.10.36/include/node/config.gypi: is not an object file
/opt/nodes/./0.10.36/include/node/nameser.h: is not an object file
/opt/nodes/./0.10.36/include/node/node.h: is not an object file
/opt/nodes/./0.10.36/include/node/node_buffer.h: is not an object file
/opt/nodes/./0.10.36/include/node/node_internals.h: is not an object file
/opt/nodes/./0.10.36/include/node/node_object_wrap.h: is not an object file
/opt/nodes/./0.10.36/include/node/node_version.h: is not an object file
/opt/nodes/./0.10.36/include/node/openssl/opensslconf.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/ngx-queue.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/stdint-msvc2008.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/tree.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/uv-bsd.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/uv-darwin.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/uv-linux.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/uv-sunos.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/uv-unix.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv-private/uv-win.h: is not an object file
/opt/nodes/./0.10.36/include/node/uv.h: is not an object file
/opt/nodes/./0.10.36/include/node/v8-debug.h: is not an object file
/opt/nodes/./0.10.36/include/node/v8-preparser.h: is not an object file
/opt/nodes/./0.10.36/include/node/v8-profiler.h: is not an object file
/opt/nodes/./0.10.36/include/node/v8-testing.h: is not an object file
/opt/nodes/./0.10.36/include/node/v8.h: is not an object file
/opt/nodes/./0.10.36/include/node/v8stdint.h: is not an object file
/opt/nodes/./0.10.36/include/node/zconf.h: is not an object file
/opt/nodes/./0.10.36/include/node/zlib.h: is not an object file
/opt/nodes/./0.10.36/lib/dtrace/node.d: is not an object file
/opt/nodes/./0.10.36/share/man/man1/node.1: is not an object file
```